### ISSUE TITLE
Remove or explain purpose of Website URL in Website properties

### ISSUE THREAD

**Issue by erikgeurts** [revive-adserver/revive-adserver#777]
Referring to #776 I would also like to propose to make the purpose of the Website URL field in the Website properties dialog clearer.
![image](https://cloud.githubusercontent.com/assets/5102461/19846801/85c05450-9f42-11e6-931f-c0cb26e383de.png)

For as long as I can remember (over a decade) this field has existed and has been mandatory, but the application doesn't really do anything with it.

As such, it might be better to either make it non-mandatory or to remove it.

**Comment by andrewatfornax** [comment]
The website URL used to be required for the old OpenX Marketplace. There is no need for it to be a mandatory field anymore.

We could either:
* Keep the field, but make it non-mandatory, and put in a note to explain this is just an informational field that might make it easier to identify the website; or
* Remove the field, because it's not used for anything.

Feedback from the community would be appreciated as to the correct approach here.

**Comment by andrewatfornax** [comment]
No feedback. 

@erikgeurts and @mbeccati - what do you think?

1. Make the field optional and add some explanatory text; or
1. Remove the field from the UI and database?

**Comment by erikgeurts** [comment]
In my experience the field triggers a significant number of support cases for our hosting business, since it creates confusion as to what the purpose is. My vote is on removing it completely, but as a second best alternative, I'd prefer to make it optional and move it to a lower position on the form so that it no longer looks to be the most important field of the form.

**Comment by mbeccati** [comment]
Removing is fine to me. As far as I can tell it is used in some text (the website spc tag generation page) and in some logic of the local invocation to check if the domain is on the same server. Such logic is broken in many cases anyway.

On the other hand we also have other unused fields on the database, e.g. website "mnemonic" - so to me we either clean them all up, or we can just hide it.

Unrelated (and maybe there's another issue?) we should make contact name and emails not mandatory too.

**Comment by erikgeurts** [comment]
@mbeccati see also #776

**Comment by andrewatfornax** [comment]
Bleh, @mbeccati, you are right, this field seems to crop up everywhere.

**Comment by erikgeurts** [comment]
I'm fine with removing it from the UI, or at least making it optional, add an explanation and move it to lower on the form. Just as long as it is more clear that it doesn't actually do anything real.

**Comment by andrewatfornax** [comment]
Agreed to hide the field for now, and remove the DB field later. However, this means we need to also prevent the generation of certain to be deprecated invocation tags (like local mode), as the Website URL field was used when generating these old tag types.
